src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/SimpleOutputTest.java

```diff
@@ -0,0 +1,276 @@
+/* (c) 2020 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.test;
+
+import static org.geoserver.complex.ComplexToSimpleOutputCommons.RULES_METADATAMAP_KEY;
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.custommonkey.xmlunit.SimpleNamespaceContext;
+import org.custommonkey.xmlunit.XMLUnit;
+import org.custommonkey.xmlunit.XpathEngine;
+import org.custommonkey.xmlunit.exceptions.XpathException;
+import org.geoserver.catalog.Catalog;
+import org.geoserver.catalog.FeatureTypeInfo;
+import org.geoserver.catalog.LayerInfo;
+import org.geoserver.catalog.MetadataMap;
+import org.geoserver.catalog.WorkspaceInfo;
+import org.geoserver.data.test.SystemTestData;
+import org.geotools.data.DataAccess;
+import org.geotools.data.FeatureSource;
+import org.geotools.data.complex.AppSchemaDataAccess;
+import org.geotools.feature.NameImpl;
+import org.geotools.jdbc.JDBCFeatureStore;
+import org.junit.Test;
+import org.opengis.feature.type.Name;
+import org.w3c.dom.Document;
+
+/** Set of tests for simple output formats on complex features requests. */
+public class SimpleOutputTest extends StationsAppSchemaTestSupport {
```

Comment:
Good! I would also check that GML3 encoding was unaffected (e..g, did not lose namespaces and the like).